roles/technical_lead.md

```diff
@@ -38,14 +37,18 @@ Technical Lead
 #### Tech direction
 
 * Advise on which tech debt the team needs to address
-* Guide team in balance of addressing tech debt vs new features - prioritize chores in the backlog
+* Role model and mentor incremental improvement strategies
+* Guide team in balance of addressing tech debt vs new features
+* Prioritize chores in the backlog
```

Comment:
It's just occurred to me that `chores` here is a little vague and carries some potential negative connotation - do we really just mean `tasks`?

Maybe I'm overthinking this though... 🤔

Reply:
I think `chores` does carry negative connotation. What about going with `tech-task` as reflected in our JIRA ticket types?

Reply:
I was wondering if this entire section could use a revamp, maybe even a removal, because it's a little MT-centric right now. I was originally thinking it might benefit from a slight re-org (this PR) and another pass later to try to clarify the responsibilities in a less platform specific way.

Reply:
I'm maybe a little less on-board with a removal as opposed to a refactor. This may be my own bias, but I'll throw out there that I think it's important to retain things like:

```
Guide team towards the simplest solution
Ensure team is breaking features into vertical slices that add value rather than doing big upfront architecture
Coach & mentor engineers in technical development & good engineering practices
```

Hopefully the spirit of these items is broadly applicable? That said - where things are too MT-specific, I'm 💯 in favor of tweaking so we can help as many get value from this doc 🎉

Reply:
Agreed, those seem broad enough. The use of chores in the backlog, for example, is less general. Want to take a pass with a followup PR?

Reply:
Yeah! That totally works for me!

Reply:
Shall we merge this - and I'll ping you + @tguidon with an edit of that section?

Reply:
Works for me! Want to practice shared ownership and merge as approving reviewer?

Reply:
Why yes! I'd be happy to - thanks for the awesome work here @adsteel